frontend/components/site/SiteSettings/AdvancedSiteSettings.jsx

```diff
@@ -116,19 +117,15 @@ export const AdvancedSiteSettings = ({
       Save advanced settings
     </button>
 
-
-    <div className="usa-alert usa-alert-warning">
-      <div className="usa-alert-body">
-        <h3 className="usa-alert-heading">Danger zone</h3>
-        <p className="usa-alert-text">Delete this site from Federalist?</p>
-        <button
-          className="usa-button usa-button-red"
-          onClick={onDelete}
-        >
-          Delete
-        </button>
-      </div>
-    </div>
+    <AlertBanner
```

Comment:
I think should make a boolean prop on `AlertBanner` for whether or not we want `role="alert"` to render.

For this "Danger zone" usage, for instance, I think we don't want that `role` present since it's not an alert being displayed in response to user action.

Here's some more reading the alert role: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role

Reply:
Seems reasonable! I think having it default to true (and be opt-out) is fine since the vast majority of our alerts are ones that we want people to know about when they happen

Reply:
agreed 👍